cmd/warm-backend.go

```diff
@@ -144,7 +144,8 @@ func newWarmBackend(ctx context.Context, tier madmin.TierConfig, probe bool) (d
 		return nil, errTierTypeUnsupported
 	}
 	if err != nil {
-		return nil, errTierTypeUnsupported
+		tierLogIf(ctx, errors.Join(err, errTierInvalidConfig))
```

Comment:
```suggestion
		tierLogIf(ctx, err)
```

We don't need the client sent error here.